cypress-tests/cypress.json

```diff
@@ -0,0 +1,14 @@
+{
+    "defaultCommandTimeout": 10000,
+    "viewportWidth": 1920,
+    "viewportHeight": 1080,
+    "baseUrl": "http://che-eclipse-che.172.19.20.205.nip.io",
+   
+    "env": {
+        "test_user_name": "admin",
+        "test_user_password": "admin",
+        "load_page_timeout": 120000,
```

Comment:
Is it nano secs or milliseconds, or secs? 
Can you reflect dimension in the variable name?